### ISSUE TITLE
Clarification regarding time series regression workflow in MAPIE v1

### ISSUE THREAD

**Issue by StijnvanderLippe** [scikit-learn-contrib/MAPIE#767]
**Is your documentation request related to a problem? Please describe.**
I would like to apply MAPIE for a time series regression project. I was looking at the documentation to understand the workflow for time series forecasting.
In the [MAPIE v1 release notes](https://mapie.readthedocs.io/en/stable/v1_release_notes.html#workflow-and-methods), the updated workflow is described. The user has to do a train, conformalization, and test set split. Next, `fit()` on training set, `conformalize()` on the conformalization set, and finally predict intervals using the `predict_interval()` method.

I am asking for clarification on the workflow for the `TimeSeriesRegressor` class. It seems like the workflow above may not be used. For instance, the `predict_interval()` method is not implemented. The[ docs](https://mapie.readthedocs.io/en/stable/generated/mapie.regression.TimeSeriesRegressor.html#mapie.regression.TimeSeriesRegressor) don't mention anything regarding the `conformalize()` method: you can call it but it's unclear what the effect is.

**Describe the solution you'd like**
It would be great if the docs could be updated to clearly indicate what the workflow is for time series regression tasks. Does that follow the v1 workflow mentioned in the release notes, or does it differ from the rest of the package?

**Comment by Valentin-Laurent** [comment]
Hello @StijnvanderLippe, as mentioned in the release notes (see [Regression and classification API changes (excluding time series)](https://mapie.readthedocs.io/en/stable/v1_release_notes.html#id6)), the changes described do not apply to TimeSeries.

Hope this helps.

**Comment by StijnvanderLippe** [comment]
Ah, that's silly of me. Somehow I overlooked the "excluding time series" part of the section header. All clear now, thanks!
